### ISSUE TITLE
0.6.x vs 0.5.x: schema is now required

### ISSUE THREAD

**Issue by mariusa** [sequelize/sequelize-auto#431]
Hi,

0.6.x doesn't work anymore unless schema is specified in options.

in auto-builder.js 
`processTables(tableResult) {`

`optables` is set to`[ { table_name: 'users', table_schema: undefined } ]`

Then this removes tables in whatever the default schema is, eg 'public'
```
lodash_1.default.intersectionWith(tables, optables, isTableEqual)
```

**Comment by steveschmitt** [comment]
Ouch, that's terrible!  Thanks for letting me know.

**Comment by steveschmitt** [comment]
Are you using the API or the command line?

**Comment by mariusa** [comment]
API

**Comment by steveschmitt** [comment]
Fixed in 0.6.2
